src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -213,6 +213,16 @@ export function uiRenderMixin(kbnServer, server, config) {
       injectedVarsOverrides
     });
 
+    // Get the list of new platform plugins. kbnServer.core is not always present in some tests.
+    // Convert the Map into an array of objects so it is JSON serializable and order is preserved.
+    const uiPlugins = kbnServer.core
+      ? [
+        ...kbnServer.core.plugins.discoveredPlugins.entries()
```

Comment:
renamed. now should be `kbnServer.newPlatform.start.plugins`

Reply:
Great, thanks!